crates/matrix-sdk/src/docs/sync_running.md

```diff
@@ -1,20 +1,11 @@
 ## Synced State
-This method waits for the event spawned by a successful API call to be synced with the SDK's state. It therefore requires a running sync to complete. This functionality is implemented for:
-+ [Client::create_room][client_create_room]
-+ [Client::join_room_by_id][client_join]
-+ [Client::join_room_by_id_or_alias][client_join_alias]
-+ [Invited::accept_invitation][invited_accept]
-+ [Left::join][left_join]
-<!--
-    These are commented out because they link to private methods.
-    + [Common::join][common_join]
-    + [Common::leave][common_leave],
--->
+This method waits for the event spawned by a successful API call to be synced with the SDK's state. It therefore requires concurrent the receival of events. One can sync events using these methods:
++ [Client::sync_once][sync_once]
++ [Client::sync][sync]
++ [Client::sync_stream][sync_stream]
++ [Client::sync_with_callback][sync_with_callback]
 
-[client_create_room]: crate::Client::create_room
-[client_join]: crate::Client::join_room_by_id
-[client_join_alias]: crate::Client::join_room_by_id_or_alias
-[common_join]: crate::room::Common::join
-[common_leave]: crate::room::Common::leave
-[invited_accept]: crate::room::Invited::accept_invitation
-[left_join]: crate::room::Left::join
\ No newline at end of file
+[sync_once]: crate::Client::sync_once
```

Comment:
Did you test whether these links actually work? I think I remember something about intra-doc links not working with `#[doc = include_str!()]`, and having to define the links outside the markdown file, as part a regular doc comment.

Reply:
Yes, hosting the docs locally, the links worked.